openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -1318,6 +1341,39 @@ def revert_changes(library_key):
         )
     )
 
+    # For each collection, trigger LIBRARY_COLLECTION_UPDATED signal and set background=True to trigger
```

Comment:
[off-topic] Out of scope, but also wanted to note that `revert_changes` might not be the best name for this function. It's described as "Discard Changes" in the UI, and it means specifically to reset the draft version to be the published version for all content. The word "revert" might be more appropriate to use when you have something that was already published, and you want to undo that publish. That's definitely a more product question though.